synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -194,6 +194,22 @@ MainWindow::init_menus()
 		sigc::ptr_fun(App::edit_custom_workspace_list)
 	);
 
+	//animation tabs
+	std::string arg_1[9] = {"move-to-tab-last","move-to-tab-1","move-to-tab-2","move-to-tab-3",
+						 "move-to-tab-4","move-to-tab-5","move-to-tab-6","move-to-tab-7","move-to-tab-8"};
+	std::string arg_2[9] = {"_(synfig-move-to-tab-last)","_(synfig-move-to-tab-1)","_(synfig-move-to-tab-2)","_(synfig-move-to-tab-3)",
```

Comment:
Can you explain what this means?

Reply:
these are just the arguments stored in an array so that the actions can be created with the loop.

Reply:
I couldn't really find t in the documentation as `Gtk::Action` as a whole seems to be deprecated and should be replaced by `Gio::Action` now. However, from what I saw in Synfigs code base so far, it's used either at times for stock ids, and other times in the _("") notation which I'm assuming then would be some sort of a label.

Reply:
> I couldn't really find t in the documentation as Gtk::Action

https://developer-old.gnome.org/gtkmm/3.24/classGtk_1_1Action.html

> and other times in the _("") notation which I'm assuming then would be some sort of a label.

Yes. And you still not see the issue here?
https://stackoverflow.com/questions/10566109/fputs-what-does-the-underscore-stand-for

Reply:
> 
> https://developer-old.gnome.org/gtkmm/3.24/classGtk_1_1Action.html

Yes, this is my go-to usually as it's the official gtkmm3 documentation. However, Gtk::Action::Create method only appears 3 times and none with the same number of arguments --2--.
![create](https://user-images.githubusercontent.com/100296264/178544724-6b733bfe-bdf3-4a5d-ae11-96a35f4b9d5e.png)
Maybe I'm somehow looking into it wrong ? i'm asking so that i can maybe avoid this in the next time.

>
>Yes. And you still not see the issue here?
https://stackoverflow.com/questions/10566109/fputs-what-does-the-underscore-stand-for

I hadn't seen this post, looking through it. it seems like this notation is necessary for being able to translate the string into other languages so then it seems like I actually have to stick to it, and not what I did here. 

update: I think now i have the get text done same as in DEFINE_ACTION hopefully this is what you mean. and thanks for the insight.

Reply:
> Yes, this is my go-to usually as it's the official gtkmm3 documentation. However, Gtk::Action::Create method only appears 3 times and none with the same number of arguments --2--.

I think this should help you: https://en.cppreference.com/w/cpp/language/default_arguments

Reply:
> 
> I think this should help you: https://en.cppreference.com/w/cpp/language/default_arguments

Yes !, that answers multiple questions actually thanks.